desktop-widgets/downloadfromdivecomputer.cpp

```diff
@@ -639,10 +640,13 @@ void DownloadFromDCWidget::enableBluetoothMode(int state)
 {
 	ui.chooseBluetoothDevice->setEnabled(state == Qt::Checked);
 
-	if (state == Qt::Checked)
-		selectRemoteBluetoothDevice();
-	else
-		ui.device->setCurrentIndex(-1);
+	if (state == Qt::Checked) {
+      		if (ui.device->currentText().isEmpty()) {
+         		selectRemoteBluetoothDevice();
+   		} else {
```

Comment:
You've moved the `else` clause from the outer `if` to the (new) inner `if` - which is not the way this should be working.

Also note that [coding style in Subsurface](https://github.com/subsurface/subsurface/blob/master/CODINGSTYLE.md) calls for tab as tabs (and not spaces), and for `if` / `else` with single statements to be without curly braces.

Reply:
Sorry about that.    I'm afraid that happened when retyping the code in git.   

FWIW, the coding style does not state that single statements go without curly braces - the only relevant point I saw was:

o both sides of an if / else clause either use or do not use curly braces:

Reply:
All good.
In essence, the purpose of having a defined coding style in a project and adhering to it is to avoid unneeded 'noise' from people reworking the style of sections of code when they are reworking or extending it. As such the 'coding style' is much more 'the style used for a majority of the code in a project' than it is a document that tries to codify this style. And in subsurface the predominant style is to not have curly braces for single `if` / `else` statements.